phraseapp/client.go

```diff
@@ -202,9 +146,45 @@ func (client *Client) send(req *http.Request, status int) (*http.Response, error
 		fmt.Fprintf(os.Stderr, "\nResponse HTTP Status Code: %s\n", resp.Status)
 	}
 
-	err = handleResponseStatus(resp, status)
+	err = handleResponseStatus(resp, expectedStatus)
 	if err != nil {
 		resp.Body.Close()
 	}
 	return resp, err
 }
+
+func paginate(u *url.URL, page, perPage int) {
```

Comment:
I would probable name that `addPagination` as I would expect `paginate` to give me something paginated.